README.md

```diff
@@ -2,6 +2,7 @@
 
 Sou o DanielHe4rt e faço lives na Twitch.tv!
 
+- 🎶 Eu gosto muito de escutar Eminem, Katy Perry e BTS.
```

Comment:
Eu acho que deveria dar uma chance para o TenaciousD

Reply:
The Beatles

Reply:
Feito